### ISSUE TITLE
Changing header size and bulleted list setups in eligibility boxes

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#32659]
## Issue Description
As part of our accessibility work, we need to change the header sizes in our featured content blocks on our benefit detail pages so that they are H2s rather than H3s. 

We also need to rework our bulleted lists and rewrite the start of each bulleted item so each line of text can stand alone (add a subject, and remove that subject from the bold lead-in line above the list). 

@lunascoop did an audit of all pages with featured content blocks, which can be found here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/10787#issuecomment-693422157

We should use this list to update all headers and any bulleted lists. 

---
## Tasks
- [x] Change H3s to H2s in eligibility boxes (Selina)
- [x] Update bulleted lists (Selina)
- [x] Review and publish (Randi)

## Acceptance Criteria
- [ ] All eligibility boxes use H2s
- [ ] All bulleted list items contain a subject and stand alone

**Comment by RLHecht** [comment]
We should hold off on this work until we know when PW is updating the template to make all the featured content Q&As H2s. Then we will only need to change any headers that are in the rich text field.

**Comment by RLHecht** [comment]
Public websites merged a PR that should automatically change any of these to H2s that use the Q&A content block. We can now make updates to any of the pages that don't use this and bring this into an upcoming sprint. FYSA @bethpottsVADEPO

**Comment by bethpottsVADEPO** [comment]
@RLHecht Have we already identified bulleted list setups that aren't in blue boxes? This could definitely be brought into an upcoming sprint.

**Comment by lunascoop** [comment]
@RLHecht I've completed updating the featured content and bulleted list setups on most of the pages mentioned in [ticket #10787](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10787). You can refer to that list for their Drupal node links. Not all of the pages needed header size or bulleted list setup changes, but there were other changes needed such as the language leading to the bulleted list (ex: "meet all of the requirements listed below" to "meet all of these requirements"), some in text links, header issues on other parts of the page, broken links, etc. 

**Here's are the pages I didn't touch**

### Life insurance
These pages were recently updated and no changes were needed

- Servicemembers’ Group Life Insurance (SGLI): http://preview-prod.vfs.va.gov/preview?nodeId=767
- Traumatic Injury Protection (TSGLI): http://preview-prod.vfs.va.gov/preview?nodeId=768

### Health care
The content on these pages are w/in a react widget and cannot be updated in Drupal. We can make changes when we add the notification reminder content in [ticket #33349](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33349#)

- VA Prescription Refill and Tracking: http://preview-prod.vfs.va.gov/preview?nodeId=27
- VA Secure Messaging: * http://preview-prod.vfs.va.gov/preview?nodeId=38
- Schedule and view VA appointments online: http://preview-prod.vfs.va.gov/preview?nodeId=37
- View your VA lab and test results online:  http://preview-prod.vfs.va.gov/preview?nodeId=40
- Get your VA medical records online:  http://preview-prod.vfs.va.gov/preview?nodeId=33 

### Disability
No changes needed to these pages:

- Disabilities that appear within 1 year after discharge: http://preview-prod.vfs.va.gov/preview?nodeId=890
- Exposure to specific environmental hazards: http://preview-prod.vfs.va.gov/preview?nodeId=889

These pages are no longer on the site and don't seem to be archived

- VA Agent Orange Registry exam: http://preview-prod.vfs.va.gov/preview?nodeId=877
- Agent Orange exposure in Vietnam or Korea: http://preview-prod.vfs.va.gov/preview?nodeId=879
- Agent Orange exposure on Navy or Coast Guard ships: http://preview-prod.vfs.va.gov/preview?nodeId=876
- Agent Orange exposure on Thailand military bases: http://preview-prod.vfs.va.gov/preview?nodeId=881
- Agent Orange exposure from C-123 aircraft: http://preview-prod.vfs.va.gov/preview?nodeId=875
- Agent Orange testing and storage outside Vietnam: http://preview-prod.vfs.va.gov/preview?nodeId=880

### Burials
No changes needed to this page

- Bereavement counseling: http://preview-prod.vfs.va.gov/preview?nodeId=854

## Other notes
I made changes to these pages, but just wanted to call some things out
- Active-duty service members and VA health care: http://preview-prod.vfs.va.gov/preview?nodeId=1005 (added to list and made updates)
- Post 9/11 GI Bill: http://preview-prod.vfs.va.gov/preview?nodeId=935 (updated featured content language but there are other changes to the page that may delay publishing (see [ticket #33033](https://github.com/department-of-veterans-affairs/va.gov-team/issues/33033#))

Let me know if you have any questions

**Comment by bethpottsVADEPO** [comment]
This is awesome, thanks so much, @lunascoop. 

@mnorthuis Is there some housekeeping we need to do re: archiving these pages that Selina identified?:

VA Agent Orange Registry exam: http://preview-prod.vfs.va.gov/preview?nodeId=877
Agent Orange exposure in Vietnam or Korea: http://preview-prod.vfs.va.gov/preview?nodeId=879
Agent Orange exposure on Navy or Coast Guard ships: http://preview-prod.vfs.va.gov/preview?nodeId=876
Agent Orange exposure on Thailand military bases: http://preview-prod.vfs.va.gov/preview?nodeId=881
Agent Orange exposure from C-123 aircraft: http://preview-prod.vfs.va.gov/preview?nodeId=875
Agent Orange testing and storage outside Vietnam: http://preview-prod.vfs.va.gov/preview?nodeId=880

**Comment by mnorthuis** [comment]
@bethpottsVADEPO Those pages were retired and redirected in Sept of 2020 ([#10768](https://github.com/department-of-veterans-affairs/va.gov-team/issues/10768)) so nothing else to be done on the Veteran facing side.  You should be good to archive them!

**Comment by RLHecht** [comment]
Thanks @lunascoop for making all these changes! I know it took a long time to get through so many pages. I've published all the ones that didn't also have other changes that we are still waiting on approval or stakeholder feedback from. I'll validate a bunch of these pages after the next deployment and then I think it's safe to close this out. I opened another ticket for us to archive the Agent Orange pages and put it in the backlog.

**Comment by RLHecht** [comment]
Work completed. Closing ticket.
